examples/addsvc/server/encode_decode.go

```diff
@@ -13,7 +13,7 @@ import (
 func DecodeSumRequest(r *http.Request) (interface{}, error) {
 	var request SumRequest
 	err := json.NewDecoder(r.Body).Decode(&request)
-	return request, err
+	return &request, err
```

Comment:
I discovered this during manual testing. Apparently it's broken on master.

    ts=2016-04-01T13:19:13Z caller=stdlib.go:89 file="http: panic serving [::1" msg="]:60046: interface conversion: interface {} is server.ConcatRequest, not *server.ConcatRequest"